examples/30_extended/study_tutorial.py

```diff
@@ -77,38 +72,18 @@
 # Creating a study is as simple as creating any kind of other OpenML entity.
 # In this examples we'll create a few runs for the OpenML-100 benchmark
 # suite which is available on the OpenML test server.
-
 openml.config.start_using_configuration_for_example()
 
-# Model that can handle missing values
-from sklearn.experimental import enable_hist_gradient_boosting
-from sklearn.ensemble import HistGradientBoostingClassifier
-
-
-# Helper functions to return required columns for ColumnTransformer
-def cont(X):
-    return X.dtypes != "category"
-
-
-def cat(X):
-    return X.dtypes == "category"
+# Model to be used
+clf = RandomForestClassifier()
 
+# We'll create a study with one run on 5 datasets present in the suite
+tasks = [115, 259, 307]
```

Comment:
I count three?

Reply:
aah missed it during re-iteration
I'll push now or wait for tests to go green first?

Reply:
I normally just push immediately.